Admin/PostAdmin.php

```diff
@@ -32,7 +31,7 @@ class PostAdmin extends Admin
     protected $userManager;
 
     /**
-     * @var Pool
+     * @var FormatterPool
```

Comment:
nope

```FormatterPool``` is used more then three times i think.

scroll down ;-)

```Pool``` just there